tools/somalier_extract.cwl

```diff
@@ -0,0 +1,32 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: somalier-extract
+doc: "Tool to convert reads/variants file into a common sites file usable by the somalier suite"
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: 1000
+    coresMin: 4
```

Comment:
Again should we have this scalable or is this just always what the program will need?

Reply:
For this guy, nope! It has no configurable resource usage